src/pretalx/orga/views/mails.py

```diff
@@ -155,8 +183,12 @@ def form_valid(self, form):
         return super().form_valid(form)
 
 
-class TemplateList(TemplateView):
+class TemplateList(PermissionRequired, TemplateView):
     template_name = 'orga/mails/template_list.html'
+    permission_required = 'orga.view_mails'
```

Comment:
Mail templates are important enough to deserve their own set of permissions, I think. So this should be `orga.view_mail_templates`, `TemplateDetail` should be `orga.edit_mail_template` (+ hide the edit button), and `TemplateDelete` should also get `orga.edit_mail_template` (+ hide the delete button).

Reply:
Done.